test/messages/commands/index.js

```diff
@@ -263,6 +264,48 @@ describe('Command Messages', function() {
     });
   });
 
+  describe('Reject', function() {
+    it('should set properties from arg in constructor', function() {
+      var message = messages.Reject({
+        message: 'tx',
+        ccode: 0x01,
+        reason: 'transaction is malformed',
+        data: new Buffer('12345678901234567890123456789012', 'hex')
+      });
+      message.message.should.equal('tx');
+      message.ccode.should.equal(0x01);
+      message.reason.should.equal('transaction is malformed');
+      message.data.toString('hex').should.equal('12345678901234567890123456789012');
+    });
+    it('should let arg be optional in constructor', function() {
+      var message = messages.Reject();
+      expect(message.message).to.be.undefined;
+      expect(message.ccode).to.be.undefined;
+      expect(message.reason).to.be.undefined;
+      expect(message.data).to.be.undefined;
+    });
+    it('should write payload correctly', function() {
+      var message = messages.Reject({
+        message: 'tx',
+        ccode: 0x01,
+        reason: 'transaction is malformed',
+        data: new Buffer('12345678901234567890123456789012', 'hex')
+      });
+      var payload = message.getPayload();
+      message = messages.Reject();
+      message.setPayload(payload);
+      message.message.should.equal('tx');
+      message.ccode.should.equal(0x01);
+      message.reason.should.equal('transaction is malformed');
+      message.data.toString('hex').should.equal('12345678901234567890123456789012');
+      //var message = messages.Reject();
```

Comment:
Looks like I forgot to take it out, will remove

-  Stein

On August 12, 2015 at 19:11:40 GMT+2, Braydon Fuller <notifications@github.com> wrote:In test/messages/commands/index.js:  > + }); > + it('should write payload correctly', function() { > + var message = messages.Reject({ > + message: 'tx', > + ccode: 0x01, > + reason: 'transaction is malformed', > + data: new Buffer('12345678901234567890123456789012', 'hex') > + }); > + var payload = message.getPayload(); > + message = messages.Reject(); > + message.setPayload(payload); > + message.message.should.equal('tx'); > + message.ccode.should.equal(0x01); > + message.reason.should.equal('transaction is malformed'); > + message.data.toString('hex').should.equal('12345678901234567890123456789012'); > + //var message = messages.Reject();   Any reason these are left commented out?  —Reply to this email directly or view it on GitHub.